app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,9 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :authentication_configuration_script_bases, :through => :configuration_scripts
```

Comment:
in general, this applies to all relations, we usually need :dependent => :destroy or :nullify everywhere